systemtest/src/main/java/io/strimzi/kafka/access/templates/KafkaTemplates.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.access.templates;
+
+import io.strimzi.api.kafka.model.kafka.KafkaBuilder;
+import io.strimzi.api.kafka.model.kafka.listener.GenericKafkaListener;
+import io.strimzi.api.kafka.model.kafka.listener.ListenerAddress;
+import io.strimzi.api.kafka.model.kafka.listener.ListenerAddressBuilder;
+import io.strimzi.api.kafka.model.kafka.listener.ListenerStatus;
+import io.strimzi.api.kafka.model.kafka.listener.ListenerStatusBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KafkaTemplates {
+
+    private KafkaTemplates() {}
+
+    public static KafkaBuilder kafkaWithListeners(String namespaceName, String clusterName, String host, List<GenericKafkaListener> listOfListeners) {
+        List<ListenerStatus> listOfStatuses = new ArrayList<>();
+
+        listOfListeners.forEach(listener -> {
+            ListenerAddress address = new ListenerAddressBuilder()
+                .withHost(host)
+                .withPort(listener.getPort())
+                .build();
+
+            listOfStatuses.add(new ListenerStatusBuilder()
+                .withName(listener.getName())
+                .withAddresses(List.of(address))
+                .build());
+        });
+
+        return new KafkaBuilder()
```

Comment:
Should we also have tests against a KRaft based cluster?

Reply:
I don't know if the Kafka Access Operator actually cares either way, so maybe we just need to raise an issue in the Kafka Access Operator to switch this over in future when KRaft is the only option.

Reply:
I wanted to add tests for KRaft as well, but I guess it would make difference if it would not be simulated - because now I'm just faking the statuses etc.

So maybe in next PRs I can add tests that will run against real clusters (ZK/KRaft) and that with the credentials provided by KafkaAccess we are able to connect to Kafka and send messages or something like that

Reply:
Yeah I think given these just fake the status and as far as I know the status of the Kafka CR is the same for KRaft and ZooKeeper clusters this is fine for now.